src/ontology/reports/sync-synonym/ncit.synonyms.confirmed.robot.tsv

```diff
@@ -517,8 +518,8 @@ MONDO:0000929	balloon cell malignant melanoma	oio:hasExactSynonym		balloon cell
 MONDO:0000929	balloon cell malignant melanoma	oio:hasExactSynonym		balloon cell malignant melanoma of skin	balloon cell malignant melanoma of skin	Balloon Cell Malignant Melanoma of Skin	NCIT:C4227	Balloon Cell Melanoma					balloon cell malignant melanoma of skin	NCIT:C4227										
 MONDO:0000929	balloon cell malignant melanoma	oio:hasExactSynonym		balloon cell malignant melanoma of the skin	balloon cell malignant melanoma of the skin	Balloon Cell Malignant Melanoma of the Skin	NCIT:C4227	Balloon Cell Melanoma					balloon cell malignant melanoma of the skin	NCIT:C4227										
 MONDO:0000929	balloon cell malignant melanoma	oio:hasExactSynonym		balloon cell malignant skin melanoma	balloon cell malignant skin melanoma	Balloon Cell Malignant Skin Melanoma	NCIT:C4227	Balloon Cell Melanoma					balloon cell malignant skin melanoma	NCIT:C4227										
-MONDO:0000929	balloon cell malignant melanoma	oio:hasExactSynonym		balloon cell melanoma	balloon cell melanoma	Balloon Cell Melanoma	NCIT:C4227	Balloon Cell Melanoma					balloon cell melanoma	NCIT:C4227										
 MONDO:0000929	balloon cell malignant melanoma	oio:hasExactSynonym		balloon cell melanoma	balloon cell melanoma	Balloon cell melanoma	NCIT:C4227	Balloon Cell Melanoma					balloon cell melanoma	NCIT:C4227										
+MONDO:0000929	balloon cell malignant melanoma	oio:hasExactSynonym		balloon cell melanoma	balloon cell melanoma	Balloon Cell Melanoma	NCIT:C4227	Balloon Cell Melanoma					balloon cell melanoma	NCIT:C4227										
```

Comment:
## Large diff because of duplicate rows
792 lines changed in the NCIT confirmed synonyms TSV. Seems to be artefact of #684.

I originally thought that the duplicate rows only appeared when there was a case difference between Mondo and the source. But this highlighted example shows a duplicate row where the rows are exactly the same.